JetStreamCompose/jetstream/src/main/java/com/google/jetstream/presentation/screens/favourites/FavouriteScreenViewModel.kt

```diff
@@ -31,18 +33,75 @@ class FavouriteScreenViewModel @Inject constructor(
     movieRepository: MovieRepository
 ) : ViewModel() {
 
-    val uiState = movieRepository.getFavouriteMovies().map {
-        FavouriteScreenUiState.Ready(it)
+
+    private val selectedFilterList = MutableStateFlow(FilterList())
+
+    val uiState = combine(
```

Comment:
NIT: Specify type explicitly for improved readability

`val uiState: StateFlow<FavouriteScreenUiState>`